src/diffusers/models/adapter.py

```diff
@@ -165,6 +167,48 @@ def forward(self, x: torch.Tensor) -> List[torch.Tensor]:
         return features
 
 
+class FullAdapter_XL(nn.Module):
```

Comment:
```suggestion
class FullAdapterXL(nn.Module): 
```

We tend to not use `_` in the class names.

Reply:
Thanks, I will modify this part.